docs/newbs_getting_started.md

```diff
@@ -93,7 +93,15 @@ Install the QMK CLI by running:
 
     python3 -m pip install --user qmk
 
-On Arch-based distros you can also try the `qmk` package from AUR (**NOTE**: this package is maintained by a community member, and at the time of writing marks some dependencies as optional that should not be):
+#### Community packages
+
+**NOTE**: These packages are maintained by a community member
+
+On Arch-based distros you can install the CLI from the official repositories:
+
+    sudo pacman -S qmk
+
+You can also try the `qmk` package from AUR (**NOTE**: this package at the time of writing marks some dependencies as optional that should not be):
```

Comment:
```suggestion
On Arch-based distros you can install the CLI from the official repositories (NOTE: at the time of writing this package marks some dependencies as optional that should not be):

    sudo pacman -S qmk

You can also try the `qmk` package from AUR:
```
Looks like it is actually the "official" package that has the dependency issues.